programs/server/config.xml

```diff
@@ -463,7 +463,8 @@
 
     <!-- Disable AuthType plaintext_password and no_password for ACL. -->
     <!-- <allow_plaintext_password>0</allow_plaintext_password> -->
-    <!-- <allow_no_password>0</allow_no_password> -->`
+    <!-- <allow_no_password>0</allow_no_password> -->
+    <!-- <allow_implicit_no_password>0</allow_implicit_no_password> -->
```

Comment:
It might be better to uncomment these three lines and write the default values here.